operations/research-and-development/product/release-process/desktop-release.md

```diff
@@ -10,168 +10,12 @@ Please refer to [the desktop app release playbook](https://community.mattermost.
-3. Docs:
-    - Finalize all documentation
-4. Dev:
-    - Publish the release in [GitHub repository](https://github.com/mattermost/desktop/releases)
-    - Merge changes made to release branch into master
-5. Marketing:
-    - Schedule Twitter announcement for 08:00 PST on the date of marketing announcement
-
-If a bug fix release is required, run through the following steps:
-
-1. Dev:
-    - Submit, review and merge patches for the release branch
-    - Cut an RC containing all bug fixes
-2. QA:
-    - Verify each of the issues in the RC are fixed  
-3. Dev:  
-    - Tag a new release (e.g. 1.1.1) and run an official build including code-signing
-    - Publish the patch release in [GitHub repository](https://github.com/mattermost/desktop/releases) with SHA-256 checksums
-    - Delete RCs after final version is shipped
-    - Merge changes made to release branch into master
-4. Release Manager:  
-    - Update [Mattermost download page](https://mattermost.com/download/)
-    - Update the download links in [installation guides](https://docs.mattermost.com/install/desktop-app-install.html)
-    - Update the changelog with notes on patch releases
-    - Draft [Mattermost Security Updates](https://mattermost.com/security-updates/) if applicable, but do not post until seven days after official release
-        - Check Security Issues spreadsheet and confirm disclosure text
-5. Marketing:
-    - Schedule Twitter announcement for 08:00 PST on the date of marketing announcement
-
-### G. (T-minus 0 working days) Release Day
-
-1. Release Manager:
-    - Post this checklist in Desktop App channel
-    - Verify all items in the last posted release checklist are complete
-    - Update the desktop app upgrade [in-product notice](https://github.com/mattermost/notices)
-    - Add new release fix versions in Jira for the next few releases
-    - Close the release in Jira
-    - Post key dates for the next release in the header of the Desktop App channel and remove links to RC candidates
-    - Check for any UserVoice feature suggestions that were completed in the current release
-    - Confirm marketing has been posted
-    - Close the release milestone in GitHub
-2. Dev:
-    - Delete RCs after final version is shipped    
-    - Check if any libraries need to be updated for the next release
-
-### H. (T-plus 10 working days) Release Updates
-
-1. Release Manager:
-    - Post this checklist in Desktop App channel
-    - Post and review [Mattermost Security Updates](https://mattermost.com/security-updates/) for the Desktop App
-    - Update Security Issues spreadsheet with issue number from posted update (e.g. v3.2.0.1)
-    - Confirm the Security Researchers list on the [Responsible Disclosure Policy](https://mattermost.com/security-vulnerability-report/) is up to date
+Note: T-minus counts are measured in "working days" (weekdays other than major holidays concurrent in US and Canada) prior to release day.
```

Comment:
I think releasing in December is fine, but I'll let @m1lt0n and @devinbinnie comment if they have additional thoughts.

Reply:
Initially that schedule was created to help spread the QA reviews of the internal QA people. Since the situation has change, we're open to a different schedule (we'd prefer to stick to the fixed cadence, but if different months e.g. Mid-April, July, October and January make more sense now, we're open to it)

@devinbinnie Any preference or adjustments we want to make here?

Reply:
Thanks @m1lt0n! Sounds good to me. @lindalumitchell thoughts?

Reply:
Thanks for asking! Given that people being out in mid- and late December might crunch testing time for a January release (possibly even more than a December release), I'd vote for either leaving it as-is, or shifting it in the other direction, e.g. Feb, May, Aug, Nov. But I'm fine either way.

Reply:
Seems like all are fine as is, and so I'm good with it as well.